README.rst

```diff
@@ -30,7 +28,7 @@ To use the Firefox Addon
 A  new directory called firefox will be created. In a new terminal window:
 
 2. cd firefox     
-3. run 'jpm run --profile your-profile'  (this runs a new instance of firefox with your default profile so your bookmarks, etc. are available)
+3. run './node_modules/.bin jpm run --profile your-profile'  (this runs a new instance of firefox with your default profile so your bookmarks, etc. are available)
```

Comment:
Did you do `npm install` again? I moved `jpm` into the `package.json` so it should install `jpm` when you do `npm install`